app/views/govuk_publishing_components/components/_cards.html.erb

```diff
@@ -0,0 +1,61 @@
+<%
+  heading ||= nil
+  items ||= nil
+  sub_heading_level = sub_heading_level || 3
+  two_column_layout ||= false
+  local_assigns ||= nil
+  shared_helper = GovukPublishingComponents::Presenters::SharedHelper.new(local_assigns)
+%>
+<% if items.present? %>
+  <div class="gem-c-cards">
+    <% if heading %>
+      <%= content_tag(shared_helper.get_heading_level, class: "gem-c-cards__heading govuk-heading-m") do %>
+        <%= heading %>
+      <% end %>
+    <% end %>
+    <ul class="gem-c-cards__list<% if two_column_layout %> gem-c-cards__list--two-column-desktop<% end %>">
+      <%
+        items.each do |item|
+
+        link = item[:link]
+
+        if !link[:path].present?
+          raise ArgumentError, "The cards component requires a href for all the links"
+        end
+
+        data_attributes = nil
+        attributes = link[:tracking_attributes] || nil
+
+        if attributes && attributes[:track_category] && attributes[:track_action]
+          data_attributes = {
```

Comment:
You're right, this is currently quite tightly coupled to the known instances where we'll be using tracking attributes (see https://github.com/alphagov/govuk_publishing_components/pull/2624/commits/5bf4e1544b808ae446d099c9df6de6f8b4423679). The other components like the select and image card seem to be more loosely defined and allow for any attributes to be passed through.

As we'd like to unblock this work and can revisit this on a future date, would it make sense for me to raise an issue about this so that we could get the PR merged?

Reply:
Yes that seems reasonable.

Reply:
Card here: https://github.com/alphagov/govuk_publishing_components/issues/2656